components/table/src/overrides/th.js

```diff
@@ -1,13 +1,14 @@
 /** @jsx jsx */
 
-import { jsx, useBrand } from '@westpac/core';
+import { jsx, useBrand, getLabel } from '@westpac/core';
 
 const Th = ({ state, ...rest }) => <th {...rest} />;
 
 const thStyles = (_, { bordered }) => {
 	const { COLORS } = useBrand();
 
 	return {
+		label: getLabel('th', { bordered }),
```

Comment:
Now I'm sure again... we should prefix them cause a cass like this `GEL-th` is not very helpful